Meeting 4 minutes (#26)
w3c/font-text-cg: minutes/meeting-202001019-4.md

> # Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC

Reviewer: ```suggestion
# Minutes of Meeting 4, 2020-10-19, 16:00–18:30 UTC
```

Corrected the date.

# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC

Attendees

Present (IRC)
liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert

> 

Reviewer: ```suggestion
```

Merge this attendance info in the IRC channel into the main section.

* Chair: Liang Hai
* Scribe: Simon Cozens

## Resolutions

None

## Actions

* Ask W3C for a dedicated video meeting space.
* Discuss meeting time scheduling on GitHub issue.
* Research setting up IRC/github-bot.
* @scottkellum, please post more notes from implementation responsive text prototypes.

## Attendance

> Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, David Corbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oğuz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren

Reviewer: ```suggestion
Andrew Glass, Bobby de Vos, Caleb Maclennan\*, Chris Chapman\*, Cosimo Lupo\*, David Corbett\*, David Singer\*, Duncan MacMichael, Greg Hitchcock, John Hudson\*, Johnathan Kew\*, Laurence Penney\*, Liang Hai\*, Mehmet Oğuz Derin\*, Nat McCully\*, Nate Willis\*, Ned Holbrook\*, Norbert Lindenberg, Peter Constable\*, Renzhi Li, Scott Kellum\*, Simon Cozens\*, Toshiya Suzuki\*, Vladimir Levantovsky\*, Zachary Scheuren\*

\* Formal participants of the FTCG.

The IRC channel recorded the following nicks: liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
```

Changes:

- Added the following names according to my Zoom screenshots: Scott Kellum, Greg Hitchcock, Peter Constable
- Normalized the following names: Anthrotype → Cosimo Lupo, Laurence → Laurence Penney, Norbert → Lindenberg
- Removed a duplication of Simon’s name.
- Marked formal attendees.
- Applied a dumb alphabetical order.
- Incorporated the IRC attendance info.

# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC

Attendees

> Present

Reviewer 2: ```suggestion
Present (IRC)
```

OR ditch this section, we probably don't need IRC vs. Video meeting attendees separately, a merged list or only the latter is probably good.

liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert

> * Chair: liang

Reviewer 2: ```suggestion
* Chair: Liang Hai
```

Either use names or IRC nicks, not both side by side in the same section. Maybe names through the attendance section, nicks for transcript?

* Scribe: Simon Cozens

## Resolutions

None

## Actions

> * Ask W3C for a dedicated meeting space.
> * Discuss time scheduling on GitHub issue.
> * Scott, please post more notes on this.
> * Try setting up the github-bot

Reviewer 2: ```suggestion
* Ask W3C for a dedicated video meeting space.
* Discuss meeting time scheduling on GitHub issue.
* Research setting up IRC/github-bot.
* @scottkellum, please post more notes from implementation responsive text prototypes.
```

## Attendance

> Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, dcorbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oguz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren

Reviewer 2: ```suggestion
Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, David Corbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oğuz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren
```

We aren't afraid of Unicode are we?
